src/components/BlogCard/index.js

```diff
@@ -21,56 +36,100 @@ const stripMarkdown = (markdown) => {
   return text;
 };
 
-export const BlogCard = ({ node, ...rest }) => (
-  <Box
-    margin={{
-      left: 'large',
-      right: 'large',
-    }}
-    pad={{
-      bottom: 'large',
-      top: 'large',
-    }}
-    gap="medium"
-    border={{
-      side: 'bottom',
-      color: 'light-2',
-      size: 'medium',
+const columns = {
+  small: ['auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto', 'auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
+};
+
+const rows = {
+  small: ['auto', 'auto', 'auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto'],
+  xlarge: ['auto'],
+};
+
+export const AllBlogsCard = ({
+  children,
+  overrideColumns,
+  overrideRows,
+  areas,
+  ...props
+}) => (
+  <ResponsiveContext.Consumer>
+    {(size) => {
+      // Take into consideration if not array is sent but a simple string
+      let columnsVal = columns;
+      if (columns) {
+        if (columns[size]) {
```

Comment:
Usually the formatter will complain about 'collapsible' logic so generally this would simplify to
```javascript
if (columns && columns[size])
```
Although since columns is hardcoded and columns will never work as a Grid columns property itself, I'd simply just get rid of columnsVal and just use columns[size] directly.

Reply:
Agree, colum[size] should be directly used.